api/v1/serializers/parks-serializer.js

```diff
@@ -0,0 +1,104 @@
+const appRoot = require('app-root-path');
+const JsonApiSerializer = require('jsonapi-serializer').Serializer;
+const _ = require('lodash');
+
+const { serializerOptions } = appRoot.require('utils/jsonapi');
+const { openapi } = appRoot.require('utils/load-openapi');
+const { apiBaseUrl, resourcePathLink, paramsLink } = appRoot.require('utils/uri-builder');
+
+const parkResourceProp = openapi.definitions.ParkResource.properties;
+const parkResourceType = parkResourceProp.type.enum[0];
+const parkResourceKeys = _.keys(parkResourceProp.attributes.properties);
+parkResourceKeys.push('owner'); // for relationships to work correctly
+const parkResourcePath = 'parks';
+const parkResourceUrl = resourcePathLink(apiBaseUrl, parkResourcePath);
+
+const getAmenitiesArray = (rawPark) => {
+  const amenitiesArray = [];
+  const listOfAmenities = ['ballfield', 'bbq', 'basketball', 'bikePaths',
+    'boatRamps', 'dogsAllowed', 'drinkingWater', 'fishing', 'hiking',
+    'horseshoes', 'naturalArea', 'dogPark', 'fields', 'shelters', 'tables',
+    'playArea', 'restrooms', 'scenicView', 'soccer', 'tennis', 'volleyball',
+  ];
+  for (let i = 0; i < listOfAmenities.length; i += 1) {
+    const amenity = listOfAmenities[i];
+    if (rawPark[amenity] === '1') {
+      amenitiesArray.push(amenity);
+    }
+  }
```

Comment:
I think you could use either `_.filter()` or `_.pickBy()` here to simplify the logic